Update hubspot-event-streaming-v2.md (#332)
amplitude/amplitude-dev-center: docs/data/destinations/hubspot-event-streaming-v2.md

!!!info "This integration is in Closed Beta"

>     This feature is in Closed Beta and is in active development. Email [integrations@amplitude.com](mailto:integrations@amplitude.com] for more information. Unlike the [HubSpot streaming integration](/data/destinations/HubSpot-event-streaming/), this integration automatically creates the necessary custom event definitions and properties in HubSpot, rather than requiring you to create them manually. 

Reviewer: ```suggestion
    This feature is in Closed Beta and is in active development. Email [integrations@amplitude.com](mailto:integrations@amplitude.com] for more information. Unlike the [HubSpot streaming integration](/data/destinations/hubspot-event-streaming/), this integration automatically creates the necessary custom event definitions and properties in HubSpot, rather than requiring you to create them manually. 
```

>     This feature is in Closed Beta and is in active development. Email integrations@amplitude.com if you are interested in getting access. Unlike the [HubSpot streaming integration](https://www.docs.developers.amplitude.com/data/destinations/HubSpot-event-streaming/), this integration will automatically create the necessary custom event definitions and properties in HubSpot, rather than requiring they be created manually. 

Reviewer: ```suggestion
    This feature is in Closed Beta and is in active development. Email [integrations@amplitude.com](mailto:integrations@amplitude.com] for more information. Unlike the [HubSpot streaming integration](/data/destinations/HubSpot-event-streaming/), this integration automatically creates the necessary custom event definitions and properties in HubSpot, rather than requiring you to create them manually. 
```

Keep these things in mind when sending events to HubSpot:

> - If you have an existing HubSpot streaming sync [HubSpot streaming integration](/data/destinations/HubSpot-event-streaming/), and you want to access these new features, select HubSpot V2 in the Data Catalog to create a new Event streaming connection . With this update, Amplitude sends events and their properties to different event definitions in HubSpot, matching each of the selected events, rather than the single event definition from the internal event name.

Reviewer: ```suggestion
- If you have an existing HubSpot streaming sync [HubSpot streaming integration](/data/destinations/hubspot-event-streaming/), and you want to access these new features, select HubSpot V2 in the Data Catalog to create a new Event streaming connection . With this update, Amplitude sends events and their properties to different event definitions in HubSpot, matching each of the selected events, rather than the single event definition from the internal event name.
```

> - If you have an existing HubSpot streaming sync [[HubSpot streaming integration](https://www.docs.developers.amplitude.com/data/destinations/HubSpot-event-streaming/), and you want to access these new functionalities, you will need to setup a new Event streaming connection by clicking on HubSpot V2 in the Data Catalog. With this update, events and their properties, will now be sent to different event definitions in HubSpot, matching each of the selected events, rather than the single event definition from the inputted internal event name.

Reviewer: ```suggestion
- If you have an existing HubSpot streaming sync [HubSpot streaming integration](/data/destinations/HubSpot-event-streaming/), and you want to access these new features, select HubSpot V2 in the Data Catalog to create a new Event streaming connection . With this update, Amplitude sends events and their properties to different event definitions in HubSpot, matching each of the selected events, rather than the single event definition from the internal event name.
```
